client/src/components/Playlist/Playlist.tsx

```diff
@@ -25,19 +26,20 @@ function Playlist() {
       if (playlists) {
         dispatch(setPlaylist(playlists)); // set all playlists from user's playlist data
       }
-      if (playlist.selectedPlaylist === '') {
+      if (selectedPlaylist === '') {
```

Comment:
Can this be refactored as follows?
```suggestion
      if (!selectedPlaylist) {
```